doc/mgmt/SONiC_OC_MGMT_Interface_HLD.md

```diff
@@ -0,0 +1,261 @@
+# Feature Name
+Openconfig support for Physical and Management interfaces via openconfig-interfaces.yang.
+# High Level Design Document
+#### Rev 0.1
+
+# Table of Contents
+  * [List of Tables](#list-of-tables)
+  * [Revision](#revision)
+  * [Scope](#scope)
+  * [Definition/Abbreviation](#definitionabbreviation)
+
+# List of Tables
+[Table 1: Abbreviations](#table-1-abbreviations)
+
+# Revision
+| Rev |     Date    |       Author       | Change Description                |
+|:---:|:-----------:|:------------------:|-----------------------------------|
+| 0.1 | 09/09/2019  |   Ravi Vasanthm     | Initial version                   |
+
+# About this Manual
+This document provides general information about openconfig support for Physical and Management interfaces handling in SONIC.
+
+# Scope
+This document describes the high level design of openconfig support for Physical and Management interfaces handling feature. Call out any related design that is not covered by this document
```

Comment:
What does this mean:
"Call out any related design that is not covered by this document" ?

Reply:
Addressed